index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 	<title>Tamagotchi Blog</title>
+	link:<style type="styles.css"></style>
```

Comment:
I think this is how it's done:

<link rel="stylesheet" type="text/css" href="styles.css">